internal/store/db/redis/store.go

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+// redis provides the Redis implementation of the StoreClient interface.
+package redis
+
+import (
+	"github.com/edgexfoundry/app-functions-sdk-go/internal/store/db"
+	"github.com/edgexfoundry/app-functions-sdk-go/internal/store/db/interfaces"
+	"github.com/edgexfoundry/app-functions-sdk-go/internal/store/models"
+)
+
+// Store persists a stored object to the data store.
+func Store(o models.StoredObject) error {
```

Comment:
Can you change the DB stubs here to do a simple mock rather than panic? Store() just return nil, RetrieveFromStore returns empty list, etc, This will allows to develop against them.